build.gradle

```diff
@@ -24,7 +24,7 @@ def dependencyVersions = [
             // rest_utils:'330' has been added to dependency temporarily for enabling Hub to send events using event emitter.
```

Comment:
can we get rid of this comment yet, or does it just need to be updated to have the right version referenced?

Reply:
Removed the hardcoded version.